x-pack/plugins/uptime/public/components/synthetics/executed_step.tsx

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiSpacer, EuiText } from '@elastic/eui';
+import React, { FC } from 'react';
+import { i18n } from '@kbn/i18n';
+import { CodeBlockAccordion } from './code_block_accordion';
+import { Ping } from '../../../common/runtime_types/ping';
+import { euiStyled } from '../../../../../../src/plugins/kibana_react/common';
+import { StepScreenshots } from './check_steps/step_expanded_row/step_screenshots';
+
+const CODE_BLOCK_OVERFLOW_HEIGHT = 360;
+
+interface ExecutedStepProps {
+  step: Ping;
+  index: number;
+  browserConsole?: string;
+}
+
+const Label = euiStyled.div`
+  margin-bottom: ${(props) => props.theme.eui.paddingSizes.xs};
+  font-size: ${({ theme }) => theme.eui.euiFontSizeS};
+  color: ${({ theme }) => theme.eui.euiColorDarkShade};
+`;
+
+const Message = euiStyled.div`
+  font-weight: bold;
+  font-size:${({ theme }) => theme.eui.euiFontSizeM};
+  margin-bottom: ${(props) => props.theme.eui.paddingSizes.m};
+`;
+
+export const ExecutedStep: FC<ExecutedStepProps> = ({ step, index, browserConsole = '' }) => {
+  const isSucceeded = step.synthetics?.payload?.status === 'succeeded';
+
+  return (
+    <>
```

Comment:
I don't think you need this extra fragment. Looks like everything is within div if I'm not mistaken.